### PR TITLE
Models: Base

### PR DESCRIPTION
**Pull request by RandyStevens**
Here I am adding to the table of contents and copying over the markdown graphs for opencannabis.geo.

### CODE REVIEW

File: 3/3-Core-Structures.md [hunk 1]
```diff
@@ -325,3 +341,289 @@ Enumerates hash algorithms supported or known by the runtime for shared use.
...
+
+## `opencannabis.geo`
+
+Location data, distance, addresses, and so on.
+
+{% nomnoml %}
+
+#fill: #d5e7ee; #8ebff2
+[Date
+  |iso8601: string]
```
Comment (RandyStevens) [inline 3/3-Core-Structures.md]: I am doing the nomnoml currently

File: 3/3-Core-Structures.md [hunk 2]
```diff
@@ -325,3 +387,1033 @@ Enumerates hash algorithms supported or known by the runtime for shared use.
...
+  |address: Address
+  |point: Point
+  |accuracy: LocationAccuracy]
+  
+[LocationAccuracy
+  |estimate: bool
+  |value: DistanceValue]
+  
+[METERS	0
+  |INCHES: 1
```
Comment (sgammon) [inline 3/3-Core-Structures.md]: this enum doesn't have a title, it should be `DistanceUnit`

File: 3/3-Core-Structures.md [hunk 3]
```diff
@@ -325,3 +387,1033 @@ Enumerates hash algorithms supported or known by the runtime for shared use.
 | SHA384 | 3 | Secure Hash Algorithm, with 384-bit width. |
 | SHA512 | 4 | Secure Hash Algorithm, with 512-bit width. |
 | MURMUR | 6 | Specifies hashing with MurmurHash. |
+
+
+## `opencannabis.geo`
```
Comment (sgammon) [inline 3/3-Core-Structures.md]: need line breaks before sections

File: 3/3-Core-Structures.md [hunk 4]
```diff
@@ -325,3 +387,1033 @@ Enumerates hash algorithms supported or known by the runtime for shared use.
...
+ | type: DeviceType
+ | flags: DeviceFlags
+ | key: DeviceCredentials]
+
+[DeviceCredentials
+ | public_key: bytes
+ | private_key: bytes
+ | sha256: string
+ | identity: string
+ | authorities: bytes: repeated]
```
Comment (sgammon) [inline 3/3-Core-Structures.md]: we should probably express repeated properties as `repeated bytes`, not `bytes: repeated`

**Review by sgammon — DISMISSED** [review]
Seriously, fantastic work dude! This is a huge improvement to the core models page.

Just need to help you with some formatting nits and we can probably get this merged today.

**Review by sgammon — APPROVED** [review]
let's review on the final pull

**Pull request merged**